src/main/kotlin/com/example/Employee/services/BankAccountService.kt

```diff
@@ -2,23 +2,22 @@ package com.example.Employee.services
 
 
 
+import com.example.Employee.controllers.request.BankAccountRequest
 import com.example.Employee.models.BankAccount
-import com.example.Employee.models.BankAccountRequest
+import com.example.Employee.models.Employee
+//import com.example.Employee.models.BankAccountRequest
 import com.example.Employee.repositories.BankAccountRepository
-import com.example.Employee.repositories.BankAccountRequestRepository
+//import com.example.Employee.repositories.BankAccountRequestRepository
 import com.example.Employee.repositories.EmployeeRepository
 import org.springframework.stereotype.Service
 import java.util.*
 import javax.persistence.EntityNotFoundException
 
 @Service
 public class BankAccountService(private val employeeRepository: EmployeeRepository,
-                                private val bankAccountRepository: BankAccountRepository,
-                                private val bankAccountRequestRepository: BankAccountRequestRepository) {
-
+                                private val bankAccountRepository: BankAccountRepository){
     public fun addAccount(employeeId: Int, account: BankAccountRequest) {
         var x = employeeRepository.findById(employeeId)
```

Comment:
Don't use variables like `x`, `a`, or `b`. Write the full name of what the variable holds.